src/app/calipso-constants.ts

```diff
@@ -1,5 +1,5 @@
 // put the logo facitility url
-export const LOGO_FACILITY = 'https://www.cells.es/logo.png';
+export const LOGO_FACILITY = 'assets/images/insert-logo-here.jpg';
```

Comment:
Minor thing, but maybe this should be just "logo.jpg" or read from environment settings?

Reply:
I was just using the code already provided.
But yes, this should be in the environments instead of a random file.

Reply:
Better to load it from environment.ts then, say "environment.logo".

Reply:
I'm doing it now